### PR TITLE
issues-109  restapi -> rebuild and task API

### PR DESCRIPTION
**Pull request by awsl-dbq**
fix conflict and Translate RESTfullAPI -> Rebuild API and RESTfullAPI -> Task API

### CODE REVIEW

File: content/en/docs/clients/restful-api/rebuild.md [hunk 1]
```diff
@@ -27,12 +27,11 @@ PUT http://localhost:8080/graphs/hugegraph/jobs/rebuild/indexlabels/personByCity
     "task_id": 1
 }
 ```
+comment：
```
Comment (javeme) [inline content/en/docs/clients/restful-api/rebuild.md:30]: "Note" is more appropriate

File: content/en/docs/clients/restful-api/rebuild.md [hunk 2]
```diff
@@ -54,11 +53,11 @@ PUT http://localhost:8080/graphs/hugegraph/jobs/rebuild/vertexlabels/person
...
 ```
 
-注：
+comment：
 
-> 可以通过`GET http://localhost:8080/graphs/hugegraph/tasks/2`（其中"2"是task_id）来查询异步任务的执行状态，更多[异步任务RESTful API](../task)
+> you can get the asynchronous job status by `GET http://localhost:8080/graphs/hugegraph/tasks/${task_id}` (the task_id here should be 2). See More [AsyncJob RESTfull API](../task)
 
-#### 1.6.3 EdgeLabel对应的全部索引重建
+#### 1.6.3 Rebulid all Indexs of VertexLabel
```
Comment (javeme) [inline content/en/docs/clients/restful-api/rebuild.md:60]: "of VertexLabel" => "of EdgeLabel"

File: content/en/docs/clients/restful-api/rebuild.md [hunk 3]
```diff
@@ -80,6 +79,6 @@ PUT http://localhost:8080/graphs/hugegraph/jobs/rebuild/edgelabels/created
 }
 ```
 
-注：
+comment：
```
Comment (javeme) [inline content/en/docs/clients/restful-api/rebuild.md:82]: ditto

File: content/en/docs/clients/restful-api/task.md [hunk 4]
```diff
@@ -6,12 +6,12 @@ weight: 13
 
 ### 7.1 Task
 
-#### 7.1.1 列出某个图中全部的异步任务
+#### 7.1.1 list all asynTasks in graph
```
Comment (javeme) [inline content/en/docs/clients/restful-api/task.md:9]: List all async tasks in a graph

File: content/en/docs/clients/restful-api/task.md [hunk 5]
```diff
@@ -6,12 +6,12 @@ weight: 13
...
 
 ##### Params
 
-- status: 异步任务的状态
-- limit：返回异步任务数目上限
+- status: the status of asynTasks
```
Comment (javeme) [inline content/en/docs/clients/restful-api/task.md:13]: ditto

File: content/en/docs/clients/restful-api/task.md [hunk 6]
```diff
@@ -129,4 +128,4 @@ PUT http://localhost:8080/graphs/hugegraph/tasks/2?action=cancel
 }
 ```
 
-此时查询 label 为 man 的顶点数目，一定是小于 10 的。
+query the number of vertex which label is man ，it must less than 10。
```
Comment (javeme) [inline content/en/docs/clients/restful-api/task.md:131]: At this point, the number of vertices whose label is man must be less than 10.

File: content/en/docs/clients/restful-api/task.md [hunk 7]
```diff
@@ -45,7 +45,7 @@ GET http://localhost:8080/graphs/hugegraph/tasks?status=success
 }
 ```
 
-#### 7.1.2 查看某个异步任务的信息
+#### 7.1.2 view the details of an asyncTask
```
Comment (javeme) [inline content/en/docs/clients/restful-api/task.md:48]: please ensure the first letter capitalized

**Review by imbajin — APPROVED** [review]

File: content/en/docs/clients/restful-api/rebuild.md (hunk 1 above)
Comment (awsl-dbq) [reply content/en/docs/clients/restful-api/rebuild.md]: okay

File: content/en/docs/clients/restful-api/rebuild.md (hunk 2 above)
Comment (awsl-dbq) [reply content/en/docs/clients/restful-api/rebuild.md]: okay

**Comment by awsl-dbq** [comment]
@javeme  Hi, please review it again, thank you.

**Review by javeme — APPROVED** [review]

File: content/en/docs/clients/restful-api/task.md [hunk 8]
```diff
@@ -93,7 +93,7 @@ DELETE http://localhost:8080/graphs/hugegraph/tasks/2
 
 #### 7.1.4 取消某个异步任务，**该异步任务必须具有处理中断的能力**
```
Comment (imbajin) [inline content/en/docs/clients/restful-api/task.md:94]: this title also need to translate, u could fix it in another **rank** pr, thanks

image

Comment (awsl-dbq) [reply content/en/docs/clients/restful-api/task.md:94]: OK fine, I will fix it in rank PR.

**Pull request merged**